src/screens/Invite/InviteScreen.tsx

```diff
@@ -1,44 +1,162 @@
+      id: "6",
+      name: "Albert",
+      head: false,
+      email: "albie@gmail.edu",
+      status: "dependent",
+    },
+  ];
+
   return (
-    <ViewContainer>
+    <View style={styles.container}>
       <Text style={styles.title1}>You're in charge!</Text>
       <Text style={styles.title2}>Invite members to Kim Family</Text>
       <ScrollView style={styles.scrollContainer}>
-        <MemberCard
-          name="Yakob Kim"
-          head={true}
-          email="yakobkimiscool@gmail.com"
-          status="hi"
-        />
-        <Pressable style={styles.backgroundContainer}>
+        {DATA.map((member) => (
+          <MemberCard
+            name={member.name}
+            head={member.head}
+            email={member.email}
+            status={member.status}
+          />
+        ))}
+        <Pressable
+          style={styles.backgroundContainer}
+          onPress={() => refRBSheet.current.open()}
+        >
           <View style={styles.addContainer}>
             <View style={styles.addCircle}>
-              <Text style={styles.add}>+</Text>
+              <SvgIcon type="invite_add" />
             </View>
-            <View style={styles.innerContainer}>
+            <View style={styles.addInnerContainer}>
               <View style={styles.rowContainer}>
                 <Text style={styles.nameText}>Add a member</Text>
               </View>
-              <Text>Invite parents and/or children</Text>
+              <Text>Invite a parent, child, or dependent</Text>
             </View>
           </View>
         </Pressable>
       </ScrollView>
       <RectangularButton
         onPress={() => navigation.navigate("LoginStack", { screen: "Invite" })}
-        text="review"
-        buttonStyle={{ backgroundColor: "#C4C4C4" }}
-        textStyle={{ color: "#525454" }}
+        text="done"
+        buttonStyle={{ backgroundColor: "#253C85" }}
+        textStyle={{ color: "#FFF" }}
       />
       <View style={styles.separator} />
-    </ViewContainer>
+      <RBSheet
+        ref={refRBSheet}
+        closeOnDragDown={true}
+        closeOnPressMask={true}
+        height={600}
+        customStyles={{
+          container: {
+            alignItems: "center",
+            borderTopLeftRadius: 20,
+            borderTopRightRadius: 20,
+          },
+        }}
+      >
+        <Text style={styles.modalTitle}>Add your family member's details</Text>
+        <FormProvider {...methods}>
+          <FormInput
+            name="name"
+            rules={{ required: "Field required" }}
+            label="full name"
+            placeholder="Firstname Lastname"
+          />
+          <FormInput
+            name="email"
+            rules={{ required: "Field required" }}
+            label="email address"
+            placeholder="email@gmail.com"
+            keyboardType="email-address"
+          />
+        </FormProvider>
+        <Text style={styles.modalText}>family relationship</Text>
+        <InviteRadioButton />
```

Comment:
Will probably need to pass a set function into this later, so that you can access the value you chose

Reply:
Just thinking ahead a bit